safe-global/web-core
# fix: update signers confirmations

## What it solves

Resolves #1374 

## How this PR fixes it
Adds `txQueuedTag` to the dependency array to fetch TransactionDetails

## How to test it
1. Create a queued transaction in a Safe with a threshold greater than 1
2. Expand the transaction in the queue so that TxSigners is visible
2. Add a confirmation and observe the TxSigners displaying the new data

## Analytics changes
N/A

## Screenshots
![Screenshot 2022-12-29 at 13 23 06](https://user-images.githubusercontent.com/32431609/209958765-cb928dcb-ad83-46fd-b7c5-ec7fb057249f.png)
